.github/workflows/tests.yaml

```diff
@@ -89,6 +89,12 @@ jobs:
           - python-version: "3.12"
             os: windows-latest
             toxenv: py312_nocover
+          - python-version: "3.13.0-rc.1"
```

Comment:
It's likely better to use a range here, like `3.13.0-rc.1 - 3.12`

Reply:
Don't think there's much difference since we'll likely want to specify new required gate names when 3.13 is out.
But, OTOH, on the plus side we'll automatically be getting new releases towards 3.13.

Reply:
yes, that's the main plus